doc/reference/provided_metrics.md

```diff
@@ -154,6 +158,31 @@ The following internal metrics are provided:
   - Number of active warnings
 ```
 
+(api-rates-metrics)=
+## API rates metrics
+
+The API rates metrics include `lxd_api_requests_completed_total` and `lxd_api_requests_ongoing`. These metrics can be consumed by an observability tool deployed externally (for example, the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack) or another third-party tool) to help identify failures or overload on a LXD server. You can set thresholds on the observability tools for these metrics' values to trigger alarms and take programmatic actions.
+These metrics consider all endpoints in the [LXD REST API](../api), plus the internal endpoints and the {abbr}`ACME (Automatic Certificate Management Environment)` challenge endpoint. See {ref}`authentication-server-certificate` for more information on ACME. They also include a label `entity_type` based on the main entity type that the endpoint is operating on. This label can have one of the following values:
+
+- `instance`
```

Comment:
@hamistao @markylaing is there somewhere in the docs we can link to for entity types, rather than having to duplicate this list? It feels like it will go out of date easily.

Also, @markylaing is the `node` entity type already exposed to end users? I thought we weren't exposing the "node" concept to end users, and using "member" or "cluster member" instead? e.g. `type ClusterGroup struct`

This feels like something we need to change otherwise we're mixing naming for the same entity type.

Reply:
I don't think there is such a list, and if there is there would still be the need to clarify that some entity types are collapsed in broader categories here. I suppose just removing this list may be an option here.